commands/kubernetes.go

```diff
@@ -1679,6 +1675,9 @@ func buildClusterCreateRequestFromArgs(c *CmdConfig, r *godo.KubernetesClusterCr
 		if r.ControlPlaneFirewall == nil {
 			r.ControlPlaneFirewall = &godo.KubernetesControlPlaneFirewall{}
 		}
+		for i := range controlPlaneFirewallAllowedAddresses {
+			controlPlaneFirewallAllowedAddresses[i] = strings.ReplaceAll(controlPlaneFirewallAllowedAddresses[i], " ", "")
```

Comment:
I'm inclined to prefer [`strings.TrimSpace()`](https://pkg.go.dev/strings#TrimSpace) which also handles all sorts of unicode variants.

Similar for the other usage elsewhere.

Reply:
Sure, I've refactored it to use strings.TrimSpace()